### PR TITLE
Military slang cheat sheet

### PR DESCRIPTION
**Pull request by stanly-johnson**
**What does your Instant Answer do?**

A Cheat Sheet defining slang terms used in the Military 

**What is the data source for your Instant Answer? (Provide a link if possible)**
Wikitionary
sourceUrl: "http://www.military.com/join-armed-forces/military-terms-and-jargon.html"

**Why did you choose this data source?**
Reliable and extensive

**Are there any other alternative (better) data sources?**
Alternative - Yes
 Better -No

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Anyone interested in military slangs 

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
Yes, https://duck.co/ia/view/military_slang_cheat_sheet

**Are you having any problems? Do you need our help with anything?**
Not Yet

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
m1
m2

-----
IA Page: https://duck.co/ia/view/military_slang_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/military-slang.json [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        "C",
+        "D",
+        "E",
+        "F",
+        "G",
+        "H"
+        ],
+    "sections": {
+        "Popular Terms": [{
+            "key":"[Bird]",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/military-slang.json]: You don't need `[` and `]` around the key value unless it contains spaces, this is why they're showing up on the cheatsheet

cc @moollaza

File: share/goodie/cheat_sheets/json/military-slang.json [hunk 2]
```diff
@@ -0,0 +1,160 @@
+{
+    "id": "military_slang_cheat_sheet",
+    "name": "Military Slang",
+    "description": "Military Slang Cheatsheet",
+    "metadata": {
+        "sourceName": "Wikitionary",
+        "sourceUrl": "http://www.military.com/join-armed-forces/military-terms-and-jargon.html"
+    },
+    "section_order": [
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/military-slang.json]: Let's use the "reference" template here: `"template_type": "reference","

**Comment by stanly-johnson** [comment]
All changes added.

**Comment by moollaza** [comment]
@stanly-johnson  this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by moollaza** [comment]
@stanly-johnson congrats! It's live: https://duckduckgo.com/?q=military+slang+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**